types/v56/types.go

```diff
@@ -603,7 +603,7 @@ type ComputeCapacity struct {
 // Description: A reference to a resource. Contains an href attribute and optional name and type attributes.
 // Since: 0.9
 type Reference struct {
-	HREF string `xml:"href,attr"`
+	HREF string `xml:"href,attr,omitempty"`
```

Comment:
In which case does a Reference return an empty HREF?

Reply:
In fact this was not for return, but for "POST"/"PUT". When attaching storage profile to catalog I wanted to use IDs (because they are what we use and having only ID is sufficient for the task). What happened is it would send ID and empty HREF field (`href=""`)which caused validation error.